sdk/identity/src/token_credentials/azure_cli_credentials.rs

```diff
@@ -94,18 +95,47 @@ mod az_cli_date_format {
     }
 }
 
+/// The respnose from `az account get-access-token --output json`.
 #[derive(Debug, Clone, Deserialize)]
-#[serde(rename_all = "camelCase")]
 struct CliTokenResponse {
+    #[serde(rename = "accessToken")]
     pub access_token: Secret,
-    #[serde(with = "az_cli_date_format")]
+    #[cfg(feature = "old_azure_cli")]
+    #[serde(rename = "expiresOn", with = "az_cli_date_format")]
     pub expires_on: OffsetDateTime,
+    #[serde(rename = "expires_on")]
+    pub expires_on_timestamp: Option<i64>,
```

Comment:
Given `expires_on` is what we want to use moving forward, can this use the name `expires_on` and rename the value that comes from `expiresOn` to something else?

Reply:
May be `local_expires_on` and `expires_on`?